### ISSUE TITLE
QA & Team Credits 

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#140]
From #112

**Comment by KatieWoe** [comment]
For https://github.com/phetsims/QA/issues/464.
A few credits need to be added to QA. So far the list includes: Megan Lai, Jacob Romero, and Kathryn Woessner

**Comment by terracoda** [comment]
In addition to QA credits, we need update other credits.

@emily-phet could you verify the following? Am I missing anyone? I added Brianna to Team.

- Software Development: Chris Malley (PixelZoom, Inc.), Taylor Want, Michael Kauzmann, John Blanco, more? 
- Team: Kelly Lancaster, Emily B. Moore, Matthew Moore, Robert Parson, Kathy Perkins, Taliesin Smith, Brianna Tomlinson?
- Sound Design: Ashton Morris

**Comment by terracoda** [comment]
@jessegreenberg, I don't know how the name ordering works for the dev list. Please adjust as needed and add names if I missed any.

**Comment by KatieWoe** [comment]
More QA additions: Jaspe Arias, Logan Bray, Liam Mulhall

**Comment by emily-phet** [comment]
@taliesin - I think this sim was started around the time Brianna was stepping away from an active design role, and was only doing interviews (and analysis and writing up of already-collected data, that didn't include this sim).  Do you remember Brianna doing interviews with this sim? If you can't recall if she was, we can probably remove her from the list.

**Comment by terracoda** [comment]
@emily-phet , I have not seen any interviews with Molarity, but I thought I saw some GT Interviews listed in the spreadsheet which is why I asked I was unsure. I will check the spreadsheet again.

**Comment by jessegreenberg** [comment]
Here are the credits now:
![image](https://user-images.githubusercontent.com/6396244/74111752-fe17b200-4b64-11ea-9311-a243f18e4287.png)

@KatieWoe @terracoda look OK?

**Comment by KatieWoe** [comment]
@jessegreenberg you didn't add the QA people from https://github.com/phetsims/molarity/issues/140#issuecomment-567669937

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, added above
![image](https://user-images.githubusercontent.com/6396244/74170962-bbed7f80-4bfb-11ea-8fb2-f9a8cc99e70f.png)

**Comment by KatieWoe** [comment]
Looks good.

**Comment by terracoda** [comment]
@jessegreenberg, I am just reaching out to Brianna.

**Comment by terracoda** [comment]
@jessegreenberg, please add Brianna Tomlinson to the Team list.

**Comment by jessegreenberg** [comment]
Added in the above commits. I think thats everyone, closing this issue.
